skrub/_reporting/_data/templates/column-associations.html

```diff
@@ -22,6 +23,7 @@
                     >
                     {{ association["cramer_v"] | format_number }}
                 </td>
+                <td> {{ association["pearson_corr"] | format_number }} </td>
```

Comment:
```suggestion
                <td>
                    {%- if not (association['pearson_corr']) | is_null -%}
                    {{ association["pearson_corr"] | format_number }}
                    {%- endif -%}
                </td>
```

maybe we could do this, so that we don't print the "nan"s


![screenshot_2025-03-31T18:28:57+02:00](https://github.com/user-attachments/assets/2abf499d-4c16-4a2f-8ad9-01fe44077615)


![screenshot_2025-03-31T18:28:50+02:00](https://github.com/user-attachments/assets/68c8c4aa-af0a-4d57-9cc6-5dc7fb46abb1)